lib/magma/project.rb

```diff
@@ -47,6 +47,20 @@ def load_project
         require_files('loaders')
         require_files('metrics')
       end
+
+      load_model_attributes
+    end
+
+    def load_model_attributes
+      model_attributes = Magma.instance.db[:attributes].
```

Comment:
Is the intent here to save database calls? Seems like it would be clearer to put this back in `Magma::Model`

Reply:
Yeah, that's why I moved it. For a project like IPI that has 17 models, doing the query upfront saves us 16 trips to the database. As a we add more projects, those queries would add up quickly. I was torn on moving it, but the trade off seemed worthwhile.